h-util-ui/prisma/schema.prisma

```diff
@@ -0,0 +1,65 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlite"
+  url      = "file:./hUtil.db"
+}
+
+model Pipeline {
+  id               Int              @id @default(autoincrement())
+  uuid             String           @unique
+  name             String
+  created          DateTime         @default(now())
+  modified         DateTime         @updatedAt
+  color            String?
+  manual_ranking   Int              @default(100)
+  pipeline_stats   PipelineStats[]
+  pipeline_modules PipelineModule[]
+}
+
+model PipelineModule {
+  id          Int @id @default(autoincrement())
+  pipeline_id Int
+  module_id   Int
+
+  pipeline Pipeline @relation(fields: [pipeline_id], references: [id], onDelete: Cascade)
+  module   Module   @relation(fields: [module_id], references: [id], onDelete: Restrict)
+
+  @@unique(name: "pipelineModuleId", [pipeline_id, module_id])
+}
+
+model Module {
+  id   Int    @id @default(autoincrement())
+  uuid String @unique
+
+  data String
```

Comment:
No `JSONB` support in sqlite, didn't feel it worth to break apart since the options objects are pretty fluid.